### DISCUSSION TITLE
Reminder - A Team Leads' Meeting Tomorrow - 4Pm UTC!

### DISCUSSION THREAD

**Discussion by Rayccheal** [clentafrica/hr-people-operations#17] [Announcements]
Dear Team Leads, @clentafrica/wg-design @clentafrica/wg-disability-gender-equality @clentafrica/wg-education @clentafrica/wg-energy @clentafrica/wg-health @clentafrica/wg-hr @clentafrica/wg-sustainability-esg @clentafrica/wg-research-and-data @clentafrica/wg-fundraising @clentafrica/wg-partnerships @clentafrica/wg-programs @clentafrica/wg-web-dev @clentafrica/wg-marketing-comms @clentafrica/wg-community-managers @clentafrica/wg-socialmedia @yigakpoa @Onodjayeke @EmmanetD 

I hope you are doing well!

This is a reminder about our Team Leads' Meeting scheduled for **tomorrow, October 26th, 2024, at 4 PM UTC.**

You can join the meeting using the following link: [Zoom](https://us05web.zoom.us/j/85319513670?pwd=PsGblvnJYDDfFEVqxi8Yhaz65ob87K.1), and to confirm your local time zone, please check [here](https://www.worldtimebuddy.com/).

Thank you, and I look forward to seeing you all tomorrow!

Best regards,
Racheal

**Comment by SatarDaud** [comment]
Please what will be there for those of us from Ghana.

**Reply by Rayccheal** [reply]
Hello @SatarDaud,

Thank you for your feedback! However, I find your question a bit unclear. Are you asking about the meeting time for Ghanaians, or is there something else you’d like to clarify?

Looking forward to your response!

Best,
Racheal

**Comment by adilkadivala** [comment]
it's nice step

**Comment by Rayccheal** [comment]
Hello everyone. This is the link to our team leads meeting happening now https://us05web.zoom.us/j/85319513670?pwd=PsGblvnJYDDfFEVqxi8Yhaz65ob87K.1

See you there!

Racheal
